docs/Documentation/run_RBM.md

```diff
@@ -1,66 +1,64 @@
 
-The file:
+After unzipping the example dataset **Salmon_River_data**, you can find the file：
 <center>**DESCRIPTION** - a text file describing the contents of each sub-directory.</center>
 
-**PS:** File **DESCRIPTION** is located in example data folder **Salmon_River_data**
-
 The sub-directories within this directory are:
 
 Sub-directories | Description
 --- | ---
-*RBM/Perl_Scripts*  | contains pre- and post-processing scripts written in Perl
-*RBM/src* | - contains the Fortran 90 source and **Makefile** that builds the executable. **RBM** and places it in the directory, **../Test_Case**. Users may wish to change this by editing the makefile.
-*RBM/rout_DA*  | - contains the source code and **Makefile** for the modified routing model, rout_RBM.
-*Salmon_River_data/Test_Case* | - contains the input files and executable for running the example problem, using input data for the Salmon River basin at a gridded resolution of 1/2° lat/long.
-*Salmon_River_data/UH_Output* | - contains the sample output from executing the routing model, **rout_RBM**, and the input data for the sample problem.
-*Salmon_River_data/VIC_Forcing* | - contains the daily precipitation (mm), max/min air temperatures (°C), and wind speed (m/sec) that provides forcings for each **VIC** grid cell.
-*Salmon_River_data/VIC_Input* | - contains the snowbands, soil, veg-param files and the world_veg_lib files
-*Salmon_River_data/VIC_Output* | - contains the meteorologic output files, full_data_lat_long, and the hydrologic output - files, flux_lat_long, from the VIC simulations.
+*RBM/Perl_Scripts*  | pre- and post-processing scripts written in Perl
+*RBM/src* | the Fortran 90 source and **Makefile** that builds the executable. **RBM** and places it in the directory, **../Test_Case**. Users may wish to change this by editing the makefile.
+*RBM/rout_DA*  | the source code and **Makefile** for the modified routing model, rout_RBM.
+*Salmon_River_data/Test_case* | the input files and executable for running the example problem, using input data for the Salmon River basin at a gridded resolution of 1/2° lat/long.
+*Salmon_River_data/UH_Output* | the sample output from executing the routing model, **rout**, and the input data for the sample problem.
+*Salmon_River_data/VIC_Forcing* | the daily precipitation (mm), max/min air temperatures (°C), and wind speed (m/sec) that provides forcings for each **VIC** grid cell.
+*Salmon_River_data/run_VIC* | the snowbands, soil, veg-param files, the world_veg_lib files and example global parameter files
+*Salmon_River_data/VIC_output* | the meteorologic output files, full_data_lat_long, and the hydrologic output - files, flux_lat_long, from the VIC simulations.
 
 ## 2. Create the RBM Executable
-Navigate to the folder, **../src**, and type:
+Navigate to the folder, **RBM/src**, and type:
 <center>**make**</center>
 
 This will create the executable, RBM, and copy a version to the folder, **../Test_Case**, where the example problem is found.
```

Comment:
I agree with you but once the executable file is generated, it can be executed as long as it is in the same environment. This is for new learners, this would be easier for them to start with.